gamemodes/terrortown/entities/entities/ttt_beacon.lua

```diff
@@ -191,11 +208,15 @@ if CLIENT then
 	end)
 
 	hook.Add("PostDrawTranslucentRenderables", "BeaconRenderRadius", function(_, bSkybox)
-		if bSkybox then return end
+		if bSkybox then
+			return
+		end
```

Comment:
This looked fine before, is it not in the limit or is the style generally enforce this structure?

Reply:
i would need to check this. But i agree that this is a questionable style.